take away the words 'at the fork'

allierowan opened the pull request:
# Issue

closes #202 

## Tasklist

 - [X] remove the works "at the fork" from the en templates
 - [X] change fixtures and test
 - [x] Add changelog entry
 - [ ] Review

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

lyzidiamond commented on languages/translations/en.json:
```diff
@@ -157,28 +157,28 @@
                 "default": "Take a sharp left at the fork",
-                "name": "Take a sharp left at the fork onto {way_name}",
-                "destination": "Take a sharp left at the fork towards {destination}"
+                "name": "Take a sharp fork onto {way_name}",
```
Take a sharp fork --> Take a sharp left

allierowan replied:
😂  good catch, LOL

lyzidiamond commented on languages/translations/en.json:
```diff
@@ -157,28 +157,28 @@
-                "name": "Take a sharp left at the fork onto {way_name}",
-                "destination": "Take a sharp left at the fork towards {destination}"
+                "name": "Take a sharp fork onto {way_name}",
+                "destination": "Take a sharp fork towards {destination}"
```
Take a sharp fork --> Take a sharp left

lyzidiamond commented on test/fixtures/v5/fork/sharp_left_destination.json:
```diff
@@ -10,7 +10,7 @@
     "instructions": {
         "de": "Scharf links abbiegen an der Gabelung Richtung Destination 1",
-        "en": "Take a sharp left at the fork towards Destination 1",
+        "en": "Take a sharp fork towards Destination 1",
```
sharp fork --> sharp left

lyzidiamond commented on test/fixtures/v5/fork/sharp_left_exit.json:
```diff
@@ -10,7 +10,7 @@
     "instructions": {
         "de": "Scharf links abbiegen an der Gabelung auf Way Name",
-        "en": "Take a sharp left at the fork onto Way Name",
+        "en": "Take a sharp fork onto Way Name",
```
sharp fork --> sharp left

lyzidiamond commented on test/fixtures/v5/fork/sharp_left_exit_destination.json:
```diff
@@ -11,7 +11,7 @@
     "instructions": {
         "de": "Scharf links abbiegen an der Gabelung Richtung Destination 1",
-        "en": "Take a sharp left at the fork towards Destination 1",
+        "en": "Take a sharp fork towards Destination 1",
```
sharp fork --> sharp left

lyzidiamond commented on test/fixtures/v5/fork/sharp_left_name.json:
```diff
@@ -9,7 +9,7 @@
     "instructions": {
         "de": "Scharf links abbiegen an der Gabelung auf Way Name",
-        "en": "Take a sharp left at the fork onto Way Name",
+        "en": "Take a sharp fork onto Way Name",
```
sharp fork --> sharp left

allierowan commented:
@lyzidiamond thanks for the catch. I fixed that, and also added the new version to the package.json and pushed tags. LMK if it looks good for merge/publish.
